agent/src/protocols/http/discovery_handler.rs

```diff
@@ -0,0 +1,79 @@
+// TODO(dazwilkin) Why is aliasing required for the lambda?
```

Comment:
is this TODO needed?

Reply:
Curious. I'd (thought I'd) had problems compiling without the aliasing, added it but was unsure why it was necessary.

Removed it all today and it appears (!?) fine.

Fixed: https://github.com/deislabs/akri/pull/135/commits/0211d62ad4be1d70a0711394771265e95c2bcb35